Apps/Common/SIMCoupledSI.h

```diff
@@ -62,6 +85,29 @@ class SIMCoupledSI : public SIMCoupled<T1,T2>
 
       if ((conv = this->checkConvergence(tp,status1,status2)) <= SIM::DIVERGED)
         return false;
+
+      // Calculate aitken acceleration factor
+      if (aitken && omega0 != 0.0) {
+        if (tp.iter > 0) {
+            Vector r1 = this->getAitkenResidual();
+            r1 -= prevRes;
```

Comment:
`Vector r1 = this->getAitkenResidual() - prevRes;` should also work.

Reply:
which yields an additional copy. granted, very minor optimization but still i prefer the current code for that reason.

Reply:
ok, keep yours